scribble-lib/scribble/acmart/acmart.tex

```diff
@@ -4,8 +4,10 @@
 % These are replaced by scribble/acmart/style.tex,
 %  which is used in combination with acmart.cls
 
+\newcommand{\SAuthorinfo}[4]{#1}
```

Comment:
My question is: since these are stubs, do they need to take any arguments or produce any output?